algos/src/sample_rate_conversion.rst

```diff
@@ -75,139 +75,138 @@ scripts print intermediate information about progress).
-“near CD quality” conversion. The int32 and float type are capable to
-provide “CD quality” and better with a higher filter spec that is
+The coefficients can be generated as int16, int24, int32, or float
+type. The type is the 3rd argument for the src_2stage (in_rates, out_rates,
+ctype) function call. It defaults to ‘int16’, which is the least memory-consuming type that provides the minimum quality. The 16-bit
+coefficients may achieve near up to a 80-90 dB stopband that will give a
+“near CD quality” conversion. The int32 and float type are capable of
+providing “CD quality” and better with a higher filter spec that is
 explained later.
 
-The SRC component is using depending on build a "tiny" int16 or "std"
-int32 coefficient set. The testbench and FW build with xtensa compiler
-defaults to 32 bit coefficients. The gcc build build for firmware
-uses 16 bit coefficients. The scripts those were use to generated them
-are src_tiny_int16.m and src_std_int32.m.
+The type of SRC component used depends on whether you are building a "tiny"
+int16 or "std" int32 coefficient set. The testbench and FW build with xtensa
+compiler defaults to 32 bit coefficients. The gcc build for firmware uses 16
+bit coefficients. The scripts those were use to generated them are
+src_tiny_int16.m and src_std_int32.m.
 
 Exclusion of non-needed conversions
 ===================================
 
-If from previous call conversion set there would be no need for
-converting from 32 to 44.1 kHz it can be achieved by adding a matrix
-with zero in the place of the non-wanted conversion. This will help
-save memory that is needed to store the conversion coefficients.
+If previous call conversion sets had no need for converting from 32 to 44.1
+kHz, it can be achieved by adding a matrix with zero in the place of the
+non-wanted conversion. This will help save memory that is needed to store
+the conversion coefficients.
 
 .. code-block:: octave
 
    >> src_generate([32e3 44.1e3 48e3],[44.1e3 48e3],[0 1; 1 1; 1 1])
 
-In the script output the removed conversion is marked with an ‘x’ and
-correspond filters are not calculated.
+In the script output, the removed conversion is marked with an ‘x’ and
+corresponding filters are not calculated.
 
 Adjustment of SRC filter specification
 ======================================
 
 The default conversions are tuned with the stopband specification to
 provide min -80 dBFs THD+N performance. The requested stop-band
-attenuation has been chosen that the THD+N criteria is met in the
+attenuation has been chosen such that the THD+N criteria is met in the
 worst-case modes.
 
-The bandwidth is about 20 kHz bandwidth for 44.1 kHz and 48 kHz sample
-rates. The bandwidth scales with minimum sample rate of
-conversion. However for rates higher than 88.1 kHz the bandwidth is
+The bandwidth is about 20 kHz for 44.1 kHz and 48 kHz sample
+rates. The bandwidth scales with a minimum sample rate of
```

Comment:
Could this be re-written "The bandwidth is scaled to correspond to the minimum sample rate of the conversion"?